app/code/Magento/Theme/view/frontend/templates/html/topmenu/renderer.phtml

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
@sshymko Agreed, as I said before "The .phtml still needs work". Is that a request for me to update it or an internal note?